web/src/lib/types/types.ts

```diff
@@ -197,6 +197,17 @@ export interface Opportunity {
   status?: string;
 }
 
+export interface QuickAction {
+  id: string;
+  name: string;
+  urlSlug: string;
+  filename: string | undefined;
+  content: string;
```

Comment:
[pebble] We should follow the same pattern as everything else and call this `contentMd` - is there a reason we're not?

Reply:
done